packages/babel-plugin-transform-runtime/src/definitions.js

```diff
@@ -68,31 +46,8 @@ export default {
       values: "object/values",
     },
 
-    Math: {
-      acosh: "math/acosh",
-      asinh: "math/asinh",
-      atanh: "math/atanh",
-      cbrt: "math/cbrt",
-      clz32: "math/clz32",
-      cosh: "math/cosh",
-      expm1: "math/expm1",
-      fround: "math/fround",
-      hypot: "math/hypot",
-      imul: "math/imul",
-      log10: "math/log10",
-      log1p: "math/log1p",
-      log2: "math/log2",
-      sign: "math/sign",
-      sinh: "math/sinh",
-      tanh: "math/tanh",
-      trunc: "math/trunc",
-      iaddh: "math/iaddh",
-      isubh: "math/isubh",
-      imulh: "math/imulh",
-      umulh: "math/umulh",
-    },
-
     Symbol: {
+      asyncIterator: "symbol/async-iterator",
```

Comment:
This may have re-broken https://github.com/babel/babel/issues/4783 by rolling back https://github.com/babel/babel/pull/5195.

(Yes, I'm very late on upgrading to Babel 7 in a few places...)